lua/packer.lua

```diff
@@ -23,7 +24,11 @@ _G._packer = _G._packer or {}
 
 -- Config
 local packer = {}
+
+---@class Config
 local config_defaults = {
+  snapshot = nil,
+  snapshot_path = util.join_paths(vim.fn.stdpath('config'), 'plugin'),
```

Comment:
I think this should probably default to somewhere in `stdpath('cache')/packer`, unless you intend for people to check their snapshots into VCS or something similar?

Reply:
Yes that was my initial idea, however it's configurable so maybe by default i'll just put use `cache`

Reply:
Although it is configurable, I think it's better to default to `cache` as we cannot really assume that users will read the docs and customize the destination before using the feature.